internal/scripts/lib/deploy.ts

```diff
@@ -85,7 +85,7 @@ export async function wranglerDeploy({
 	sentry?: {
 		authToken: string
 		project: string
-		release?: string
+		environment?: string
```

Comment:
shouldn't this still be called `release` ? environments are a different thing in sentry

Reply:
I guess the release is the combination of an environment and a random string to specify which release exactly it is. And that what was the `release` prop used to hold, both of these values. But now we just send the first part of a relase.

And it seems like we do then [forward this value](https://github.com/tldraw/tldraw/blob/2f6593e71b5f5a4ed1d7db92cd0bf6394db02cfa/packages/worker-shared/src/sentry.ts#L32) as `environment` when we report issues.